build-tools/helm/spark-kubernetes-operator/values.schema.json

```diff
@@ -0,0 +1,712 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "description": "Values for Spark Kubernetes Operator Helm Chart",
+  "properties": {
+    "image": {
+      "type": "object",
+      "description": "Operator image configuration",
+      "required": [
+        "repository",
+        "pullPolicy",
+        "tag"
+      ],
+      "properties": {
+        "repository": {
+          "type": "string",
+          "description": "Docker image repository"
+        },
+        "pullPolicy": {
+          "type": "string",
+          "description": "Image pull policy",
+          "enum": [
+            "IfNotPresent",
+            "Always",
+            "Never"
+          ]
+        },
+        "tag": {
+          "type": "string",
+          "description": "Docker image tag"
+        },
+        "digest": {
+          "type": "string",
+          "description": "Docker image digest, takes precedence over tag if set"
```

Comment:
`tag` is required above. But when `digest` is provided, we don't need `tag`?

Reply:
Yes,  right. `tag` will be ignored when digest is provided. Please see here, @viirya .

https://github.com/apache/spark-kubernetes-operator/blob/661757ae256dec8444e938ebae6b0a4d378d5b9d/build-tools/helm/spark-kubernetes-operator/values.yaml#L19-L21

Reply:
If both are set, `tag` will be ignored semantically, but `tag` field is still required.